plutus-core/plutus-core/test/Names/Spec.hs

```diff
@@ -0,0 +1,274 @@
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE TypeFamilies          #-}
+
+module Names.Spec where
```

Comment:
Extracted names-related tests into their own spec from the messy main spec file.